### DISCUSSION TITLE
How to stop receiving pre-releases?

### DISCUSSION THREAD

**Discussion by threo9** [AdguardTeam/AdGuardHome#4992] [Q&A] [answered]
Please tell me how to make it.
I've tried to find it in the document. If it mentioned this, sorry for the language barrier.

**Comment by ainar-g** [comment, marked as answer]
You need to:

1.  backup your AdGuard Home data;
2.  download the release archive from the [releases page][1];
3.  stop AGH;
4.  replace the old `AdGuardHome` binary with the one that you have extracted from the archive;
5.  restart AGH;

On some OSs, you'll also need to refresh the daemon configuration.  For example, with systemd you might need `systemctl daemon-reload`.

[1]: https://github.com/AdguardTeam/AdGuardHome/releases

**Reply by threo9** [reply]
Do I need to modify the configuration file when I use an old version of the application?

**Reply by EugeneOne1** [reply]
@Suansuan-Keai, only the minor part of updates changes the configuration schema.  Since the `v0.107.0` we reflect all the changes of the configuration file in the [CHANGELOG.md](https://github.com/AdguardTeam/AdGuardHome/blob/master/CHANGELOG.md).  So the manual changing of the file is only required if there is the **Configuration Changes** section exists within any release between the current one and the one you want to rollback to.

**Reply by threo9** [reply]
I see, thank you for your response!
